app/views/admins/members/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= link_to "New Member", new_admin_member_path %></h1>
+
+<table border="1">
+  <% @members.each do |member| %>
+    <tr>
+      <td><%= member.last_name %></td>
+      <td><%= member.first_name %></td>
+      <td><%= member.email %></td>
+      <td><%= button_to "View", admin_member_path(member), method: :get %></td>
+      <td><%= button_to "Edit", edit_admin_member_path(member), method: :get %></td>
```

Comment:
Oh interesting....I didn't see that. Actually from the docs it looks like `button_to` is used to avoid having search bots trigger the link. If you're just trying to get a link that looks like a button it's better to use `link_to` with `class: 'btn'`